test/axes/categoryAxisTests.ts

```diff
@@ -177,4 +177,19 @@ describe("Category Axes", () => {
 
     svg.remove();
   });
+
+  it("_computed space variables should be set on requestedSpace", () => {
```

Comment:
This test name seems wrong? We're checking that the variables are set correctly on `computeLayout()`, right?